core/storage/hamt/hamt.hpp

```diff
@@ -125,7 +137,7 @@ namespace fc::storage::hamt {
     outcome::result<void> remove(Node &node,
                                  gsl::span<const size_t> indices,
                                  const std::string &key);
-    outcome::result<void> cleanShard(Node::Item &item);
+    static outcome::result<void> cleanShard(Node::Item &item);
     outcome::result<void> flush(Node::Item &item);
     outcome::result<void> loadItem(Node::Item &item);
```

Comment:
Can be marked 'const'
```suggestion
    outcome::result<void> loadItem(Node::Item &item) const;
```